### ISSUE TITLE
Send email if no issue

### ISSUE THREAD

**Issue by lashnag** [Hopebaytech/redmine_mail_reminder#76]
Hello! I have email if there is no issues:

Project "certs" with query "п≈п╟п╢п╟я┤п╦ п╢п╩я▐ п╬п╠п╫п╬п╡п╩п╣п╫п╦я▐"  [0;94;49m	 is processing. [0m

**Comment by lashnag** [comment]
Environment:
  Redmine version            3.2.3.stable
  Ruby version                  2.2.5-p319 (2016-04-26) [amd64-freebsd10]
  Rails version                  4.2.7
  Environment                  production
  Database adapter          Mysql2

And i see in code comment:

# Not Sending email if there are no issues

**Comment by lashnag** [comment]
![1](https://cloud.githubusercontent.com/assets/5690099/17210910/a8f3b7be-54ce-11e6-8686-6f3464e8d9e6.png)

**Comment by jcppkkk** [comment]
@lashnag  do you see issues when you search the query in redmine?

**Comment by lashnag** [comment]
![2](https://cloud.githubusercontent.com/assets/5690099/17217621/a27ac818-54ed-11e6-8402-1cb5d9bb0c3a.png)
Yes, but i don't see any addiction

**Comment by lashnag** [comment]
https://github.com/Hopebaytech/redmine_mail_reminder/blob/master/app/models/mail_reminder_mailer.rb - in this file on 43 i can see check if empty, but i don't khow ruby...

**Comment by jcppkkk** [comment]
@lashnag yes redmine_mail_reminder will not send mail if the query result has no issue, to prevent user received empty mail with no content. So you need to check if your user(the one want to receive mail) can see any issues in the query page on redmine. if answer is no then mail will not sent.

**Comment by lashnag** [comment]
Now i set reminder only to send mail to administrators:

Project "certs" with query "Cert issue reminder"  [0;94;49m	 is processing. [0m
v.egorov@platron.ru
a.lashnev@platron.ru

But the problem is steel there...
![1](https://cloud.githubusercontent.com/assets/5690099/17286723/fc0fe064-57d3-11e6-95db-60c7abaa7240.png)

**Comment by lashnag** [comment]
I can log some queries if you say where...

**Comment by lashnag** [comment]
@jcppkkk do you have any ideas? I can log anything...
